### DISCUSSION TITLE
Server and agent versions are not the same

### DISCUSSION THREAD

**Discussion by bravery-only** [Ylianst/MeshCentral#5830] [General]
My server is version 1.1.21 
and
Agents version 1.1.18
Is it necessary to update the agents? Is it possible that the connection between the agent and the server will be interrupted due to a different version?
​

**Comment by si458** [comment]
Where are u seeing `agents version?`
do u mean the agent core version, as that should be something like 2022/2023?

**Reply by bravery-only** [reply]
What I mean is that the Mesh Central server version and the Mesh Agent version are different.
Do I have to update the mesh agents at the same time as the server update?

**Reply by si458** [reply]
we haven't released a meshagent update since 2022/2023,
But if u have the default config.json, it will auto update the remote agent when a new meshagent is released :)

**Reply by bravery-only** [reply]
I have disabled the automatic update of mesh agent in the config file. Do I need to Enable it?

**Reply by si458** [reply]
No it's a personal choice.
by default it will autoupdate the meshagents if a new version is found,
But if u disable the autouodate of the meshagents, then u can manually update each agent/group by sending the console command `agentupdate`

**Reply by bravery-only** [reply]
If mesh agent has been updated since 2022/2023, then it is better to disable the automatic update, right?

**Reply by si458** [reply]
Again it's a personal choice!
Like I am in the process of making new builds of the meshagent which include fixes over the years
